src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -300,7 +303,7 @@ object MarathonSchedulerActor {
 
   case object ReconcileHealthChecks
 
-  case class ScaleApp(appId: PathId) extends Command {
+  case class ScaleApp(appId: PathId, force: Boolean = false) extends Command {
```

Comment:
Please document the semantics of the `force` parameter here.  The other commands are also lacking but let's ++docblocks going forward.